### PR TITLE
fix: Statuses breaking line in the UI

### LINKED ISSUE #1982: Flexbox improvement checklist on the workspace page
## What is your suggestion?
There are a couple of flexbox improvements that we can make to the workspace page:

- [ ] As discussed in #1963, reconsider the spacing on the tables such that all the status columns align neatly (and preferably under the action buttons.
- [ ] On smaller screens, we are having difficulties with a two column layout (see #1953). Even with that aforementioned ticked resolved, we still get wrap on the status columns:
Screen Shot 2022-06-02 at 10 03 50 AM
I think the best way to handle this would just be to create two rows at a certain resolution. A quick mock in Figma is probably warranted. 

## Are there any workarounds to get this functionality today?
Screen is totally usable as is

## Are you interested in submitting a PR for this?
Sure

### PR DESCRIPTION
**Pull request by BrunoQuaresma**
Closes #1982

### CODE REVIEW

**Review by kylecarbs — APPROVED** [review]
Going to check out the storybook when ready, but based on images shared in Discord this looks great!

**Review by jsjoeio — APPROVED** [review]
Code changes look good 👍🏼

request: include screenshots for UI changes so that I don't have to pull locally or search through Chromatic

File: site/src/components/BuildsTable/BuildsTable.tsx [hunk 1]
```diff
@@ -79,7 +79,9 @@ export const BuildsTable: FC<BuildsTableProps> = ({ builds, className }) => {
                   </span>
                 </TableCellLink>
                 <TableCellLink to={buildPageLink}>
-                  <span style={{ color: status.color }}>{status.status}</span>
+                  <span style={{ color: status.color }} className={styles.status}>
```
Comment (Kira-Pilot) [inline site/src/components/BuildsTable/BuildsTable.tsx:82]: Now that this has a class, we can move the inline style into that class.

Comment (BrunoQuaresma) [reply site/src/components/BuildsTable/BuildsTable.tsx:82]: It is inline because it is dynamic and passed from the props.

**Comment by BrunoQuaresma** [comment]
@kylecarbs these changes are not related to the Discord ones. It only avoids the status icon and the "verb" getting split into two lines on small viewports.

**Comment by BrunoQuaresma** [comment]
@jsjoeio totally, since it was just a tiny small thing I skipped, but I will do it next time, my bad.

**Comment by kylecarbs** [comment]
@BrunoQuaresma ahh my bad, I just assumed 🙈

File: site/src/components/BuildsTable/BuildsTable.tsx (hunk 1 above)
Comment (Kira-Pilot) [reply site/src/components/BuildsTable/BuildsTable.tsx:82]: we can still pass it i think, but nbd if you want to leave it inline!

**Review by Kira-Pilot — APPROVED** [review]

**Pull request merged**